packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Button.doc.ts

```diff
@@ -15,6 +19,12 @@ const data: ReferenceEntityTemplateSchema = {
   ],
   category: 'Components',
   related: [],
+  defaultExample: {
+    codeblock: generateCodeBlockForButton(
+      'Render a button that presents a toast',
+      'default.example',
+    ),
+  },
```

Comment:
I think you'll want to add a `definition` for the ButtonType too

Reply:
Done 👍